python/perf-kernels/06-fused-attention-fwd-transV.py

```diff
@@ -18,6 +18,9 @@
 import triton
 import triton.language as tl
 
+# Pick the fp8 data type
+#float8:tl.constexpr = torch.float8_e5m2fnuz
+float8:tl.constexpr = torch.float8_e4m3fnuz
```

Comment:
Both e5m2 and fp16 (e5m10) have almost the same dynamic range. But e4m3 maxes out at 448 vs 64k for the other two. This would require scaling. It might be worth adding that either here or as a future todo (scaling is just dividing all larger precisions by a value to reduce their range, and then multiplying the result of a GEMM by that number to expand it back).

Reply:
Just want to clarify. There is no need for scaling when using e5m2?

Reply:
Yes e5m2 shouldn't require scaling when converting from fp16 but will require scaling if converting from bf16.

Reply:
More specifically, general `e5m2` has same exponent (e5 with bias 15) as fp16, so no scaling is needed. AMD `e5m2b16` uses a different bias 16, so `e5m2b16` has a smaller range than the general `e5m2`, so, theoretically, there is possibility of overflow.